packages/block-library/src/query/utils.js

```diff
@@ -375,7 +379,14 @@ export const useUnsupportedBlocks = ( clientId ) => {
 			getClientIdsOfDescendants( clientId ).forEach(
 				( descendantClientId ) => {
 					const blockName = getBlockName( descendantClientId );
-					if ( ! blockName.startsWith( 'core/' ) ) {
+					const blockInteractivity = getBlockSupport(
+						blockName,
+						'interactivity'
+					);
+					if (
+						! blockName.startsWith( 'core/' ) &&
+						! blockInteractivity
```

Comment:
We may need to revisit this conditional and allow all blocks with supports interactivity true and/or interactivity.clientNavigation.